src/test/unittest.ts

```diff
@@ -2,9 +2,13 @@
 require('source-map-support').install();
 
 import apiCompatibilityChecks from './api_compatibility';
+import mocktracerImplementationTests from './mocktracer_implemenation';
 import noopImplementationTests from './noop_implementation';
 import opentracingAPITests from './opentracing_api';
 
+// basic tests for the mock tracer
+mocktracerImplementationTests ();
```

Comment:
running apiCompatibilityChecks on MockTracer would be more useful

Reply:
Will port them over. Makes sense